### ISSUE TITLE
Add support for Bower package management

### ISSUE THREAD

**Issue by Siyfion** [uxebu/bonsai#213]
Currently the bower links directly to the github repo, which means that after getting the latest version it first has to be built before being usable. This goes against the idea of bower containing ready-to-use packages.

So could we look into adding a `bower.json` file, and including a pre-built version in the tagged releases?

**Comment by StrictlySkyler** [comment]
+1

**Comment by Siyfion** [comment]
@basecode Do you have any thoughts on this?

**Comment by basecode** [comment]
Hi @Siyfion, thank you for your feedback!

Providing a ready-to-use package sounds doable. I'd like to see how bower is integrated in other projects first. Could you point me to a good project please?

**Comment by Siyfion** [comment]
Of course, SugarJS recently extended their bower support to the latest specs, so this project and `bower.json` is a good point of reference: https://github.com/andrewplummer/Sugar/blob/master/bower.json

**Comment by earaujoassis** [comment]
+1

**Comment by JZ-at-TP** [comment]
+1

**Comment by lyrobinson** [comment]
+1

**Comment by kibiz0r** [comment]
+1

From browsing around, it looks like there are two general strategies here:
1. Keep a copy of the minified release in the repo
2. Make a separate repo that is just for the bower distribution

I'll probably end up forking and doing option 1 if I keep using rails-assets on my project. I'll submit a pull if that's the case, but long-term this would have to be maintained by the devs.

**Comment by wjdp** [comment]
:+1: 

Also a note for people needing a solution now: `bower install --save https://github.com/uxebu/bonsai/releases/download/v0.4.5/bonsai-0.4.5.js` will install the latest (at time of writing) version and save to your bower.json file.
